spec/services/target/create_target_services_spec.rb

```diff
@@ -40,5 +40,61 @@
         expect { subject }.to raise_error(MaxAllowTargetsError)
       end
     end
+
+    context 'when two targets match' do
+      let!(:target) do
+        create(:target,
+               latitude: target_params[:latitude],
+               longitude: target_params[:longitude],
+               topic_id: topic_id)
+      end
+
+      it 'creates a match' do
+        expect { subject }.to change { Match.count }.from(0).to(1)
+      end
+    end
+
+    context 'when two targets do not match' do
+      context 'when they have a different topic' do
+        let!(:target) do
+          create(:target,
+                 latitude: target_params[:latitude],
+                 longitude: target_params[:longitude])
+        end
+
+        it 'does not create a match' do
+          expect { subject }.not_to change { Match.count }
+        end
+      end
+
+      context 'When is out of range' do
```

Comment:
Try to be consistent with the casing (always start the description with a lower casing)